src/style.css

```diff
@@ -1,3 +1,77 @@
+:root {
+  --accent-color: #00acdc;
+  --bg-color: white;
+  --text-color: black;
+  --header-size: 60px;
+}
+
 body {
+  padding: 0;
+  margin: 0;
+}
+
+.header {
+  width: 100%;
+}
+
+.header__container {
+  margin: 0 auto;
```

Comment:
Be consistent with your *vertical* margins (Add only top or only bottom margin, but not both). Vertical margins can be collapsed in some cases. Here you're using margin: 0 auto; which applies both vertical and horizontal margins. Try to avoid this if possible.